tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -683,6 +683,11 @@ def canvas_sections_on(context):
     context.canvas_sections_enabled = True
 
 
+@pytest.fixture
+def canvas_groups_launch(context):
+    context.canvas_is_group_launch = True
```

Comment:
I didn't manage to do this local to the test (ie change context directly on the test)